repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -318,6 +318,10 @@ object HttpServletRequestWrapper {
     parsedParameterMap.toMap
   }
 
+  def parseQueryString(queryString: String): java.util.Map[String, Array[String]] = {
```

Comment:
So i can do the null check in the other function, but it's not strictly necessary there, because of some craziness from boxing and unboxing of types. Or at least that's what i was seeing.